src/pages/ImportData.js

```diff
@@ -38,23 +38,26 @@ const styles = {
   }
 }
 
-const ChoiceButton = ({ iconName, title, detail, style, ...props }) => {
+const ChoiceButton = ({ iconName, title, detail, style, onClick, disabled, ...props }) => {
+  const color = disabled ? colors.dark(0.25) : colors.accent(1)
   return h(Clickable, {
     style: {
       ...style,
       padding: '1rem', marginTop: '1rem',
       display: 'flex', alignItems: 'center',
-      border: `1px solid ${colors.accent(1)}`, borderRadius: 4
+      border: `1px solid ${color}`, borderRadius: 4,
+      cursor: disabled ? 'not-allowed' : 'pointer'
     },
-    hover: { backgroundColor: colors.accent(0.1) },
+    hover: disabled ? undefined : { backgroundColor: colors.accent(0.1) },
+    onClick: !disabled && onClick,
     ...props
   }, [
-    icon(iconName, { size: 29, style: { flex: 'none', marginRight: '1rem', color: colors.accent(1) } }),
+    icon(iconName, { size: 29, style: { flex: 'none', marginRight: '1rem', color } }),
     div({ style: { flex: 1 } }, [
-      div({ style: { fontWeight: 'bold', color: colors.accent(1) } }, [title]),
-      div([detail])
+      div({ style: { fontWeight: 'bold', color } }, [title]),
+      div({ style: disabled ? { color: colors.dark(0.25) } : undefined }, [detail])
```

Comment:
The disabled could move in if we want.

Reply:
it seems odd to duplicate the `disabled ? colors.dark(0.25)` here and on line 42. But it can't be reused since it's `undefined` here and `colors.accent(1)` above?

Reply:
Exactly - in order to have the text be default, rather than accent for the details (which I think we want), we need a separate var